seb_openedx/seb_keys_sources.py

```diff
@@ -0,0 +1,35 @@
+"""
+SEB KEYS FETCHING
+Available functions that can be used to fetch Secure Exam Browser keys
+"""
+from django.conf import settings
+from django.utils import six
+
+
+def from_other_course_settings(course_module):
+    """
+    ENABLE_OTHER_COURSE_SETTINGS must be enabled for this option to work,
+    meaning Open edX version >= release-2018-08-07-11.59 (e.g. Hawtorn doesn't work)
+    """
+    if hasattr(course_module, 'other_course_settings'):
+        other_settings = course_module.other_course_settings
+        return other_settings.get('seb_keys', None)
+    return None
+
+
+def from_global_settings(course_module):
+    """
+    Retrieve from global settings, for example:
+    # lms/env/common.py
+    SEB_KEYS = {
+        "course-v1:edX+DemoX+Demo_Course": ["FAKE_SEB_KEY"]
+    }
+    """
+    if hasattr(settings, 'SEB_KEYS'):
+        course_id = six.text_type(course_module.id)
```

Comment:
it is a lot of overkill to require the course_module if we are only using the id.
The same is going to happen once we create a seb_key_source that reads from a model.

What about we pass only the course_id [here](https://github.com/eduNEXT/seb-openedx/pull/3/files#diff-a9302b91a0c07ba15f2bb46db396f7bdL28) and we read the actual module in `from_other_course_settings`